apps/opik-frontend/src/components/pages/TracesPage/TracesSpansTab/TracesSpansTab.tsx

```diff
@@ -585,7 +629,32 @@ export const TracesSpansTab: React.FC<TracesSpansTabProps> = ({
           statisticKey: `${COLUMN_FEEDBACK_SCORES_ID}.${label}`,
         }) as ColumnData<BaseTraceData>,
     );
-  }, [dynamicScoresColumns]);
+
+    // Add span feedback scores columns only for traces
+    if (type === TRACE_DATA_TYPE.traces) {
+      const spanFeedbackScoresColumns = dynamicSpanScoresColumns.map(
+        ({ label, id, columnType }) => {
+          // Extract the score name without the "(span)" suffix for matching
+          const scoreName = label.replace(" (span)", "");
```

Comment:
I’d put the ` (span)` suffix into a constant and reuse it both when creating and when processing. Or better, create functions that format a column name and parse it back out